src/CppUTest/TestFailure.cpp

```diff
@@ -395,6 +395,7 @@ UnexpectedExceptionFailure::UnexpectedExceptionFailure(UtestShell* test)
 
 static SimpleString getExceptionTypeName(const std::exception &e)
 {
+#if defined(_CPPRTTI) || defined(__GXX_RTTI)
```

Comment:
Ok. I'll merge this... but these defines will need to go to the CppUTestConfig and not here like this. This will not be sustainable on the long run.

Reply:
Ok, made changes via #1671.